### ISSUE TITLE
Usage in integration test

### ISSUE THREAD

**Issue by theoreticalsampling** [ayalma/feature_discovery#57]
Is there any way, I can interact with the DescribedFeatureOverlay and/ or its contents in an integration test? I try to get Widgets under tapTarget and description using `find.byValueKey` but I don't get any results.

**Comment by matejkramny** [comment]
In addition using `enablePulsingAnimation` blocks `pumpAndSettle()`.

The driver turns off animations, is there not a way to respect it?

**Comment by japatel225** [comment]
Hi @theoreticalsampling @matejkramny , did you find a way to find and tap on the `tapTarget` widget?

**Comment by sxtfv** [comment]
+1, @theoreticalsampling @japatel225 did you succeed here?

**Comment by theoreticalsampling** [comment]
No, I ended up disabling DescribedFeatureOverlay in my integration tests
